packages/react-native/android/src/main/java/backtrace/library/BacktraceAndroidBackgroundUnhandledExceptionHandler.java

```diff
@@ -0,0 +1,106 @@
+package backtraceio.library;
+
+import android.content.Context;
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+
+import com.facebook.react.bridge.Callback;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.module.annotations.ReactModule;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+/**
+ * Handle unhandled Android exceptions from background threads.
+ */
+@ReactModule(name = backtraceio.library.BacktraceAndroidBackgroundUnhandledExceptionHandler.NAME)
+public class BacktraceAndroidBackgroundUnhandledExceptionHandler extends ReactContextBaseJavaModule implements Thread.UncaughtExceptionHandler  {
+    private final static transient String LOG_TAG = BacktraceAndroidBackgroundUnhandledExceptionHandler.class.getSimpleName();
+
+    private Thread.UncaughtExceptionHandler mRootHandler;
```

Comment:
Can we not do Hungarian notation?

Reply:
This is the style from the java code. I would prefer to stick to java code convention rather than using javascript code style here

Reply:
After revisiting the code I adjusted it to your suggestions! Thank you!